Makefile

```diff
@@ -1,7 +1,11 @@
 # Utility docker image to generate Go files from .proto definition.
-# https://github.com/infobloxcto/atlas-gentool
+# https://github.com/infobloxopen/atlas-gentool
```

Comment:
Looks like this diff shows changes that are already in `master`. Wonder why this may happen

Reply:
I did a rebase between commits. Sometimes that results in oddities like this. If I were more proficient with git I might have more insight to offer on why this is a 'diff'